libs/ledgerjs/packages/cryptoassets/src/data/esdt.json

```diff
@@ -1 +1 @@
e6dabebfe72e6dc3759a","AshSwap",false],["BHAT","424841542d633166646533",18,"30450221009bd94edb228d0810881b22f3e89a2d5617077ade5329e5ff2d9332cd179beb7f02205f15419fa374050b8d0dc38182ccf310e8f6cf8ea7a79eac64a5e20eec28ff7f","BHNetwork",false],["BUSD","425553442d343062353765",18,"3044022028c5f58c74c99b905fe0d12bbd431d37f3ba395741c9f92cffedc1954b5fc9b6022076fd13b7fd1d193c038b6eab3662b3a1572db4ddc805d3c6369c4f1c8a86d618","WrappedBUSD",false],["CRU","4352552d613566346161",18,"304502210097d5954f6a302a2ad51dee64a6d80b98299d96fb2bd8bcaf72e94dd932fc13aa0220722b509211f28d334d8eafae96521d8550ed5c74743383832ce8436353310386","CrustNetwork",false],["ISET","495345542d383465353565",6,"304402201cb6ecc0a50e93cac00a0ba06f8a07efb04803aeb2bf748efd5078efe3b79f8b02204013c555b35f09d4a87ca30483a4f1e996554463c8354cfd80e93d09d4140a00","IsengardToken",false],["ITHEUM","49544845554d2d646636663236",18,"304402202a59477deed3f19e4abd33be2c857f45cddc5f8020853816e7ff72d288db94c102200a9f9309dbeb9ede330299f9faf950d39a1e5ab1e1c14d5c046403bb9af6602a","Itheum",false],["MEX","4d45582d343535633537",18,"304402207c9727bf3a0af254052154bceefe835856e4a7f6538cb6f5c3ef12422c6ab5dd02207d7a6c88777032079cd57b5b859a6e8dee4b73bb4fa699f0340f99fb70080a8d","MEX",false],["QWT","5157542d343661633031",6,"3045022100e43c1ad7d3e4a5439e2ff7b97bb7901ca1a36a130b0e6e74464d65cc8d282bbb02206d318a05ff0dde6148856fc8391f1878cbdea591db47a74ad117a6a821e835f2","QoWatt",false],["RIDE","524944452d376431386539",18,"3045022100852cc03c62a28effb3d6a8f21b6ec6d8464a9d9e28f42c45c9d4a9492bdf19bb022041c5ebdf53f1369b93523b222c62be14bc95c67d30a1fdf9834ef287447e4b79","holoride",false],["SFIT","534649542d616562633930",18,"304402204abef32b5df9129086d14d8130f0658085a30b061f39d8226e8a2e2223ea7b5b02207bf2f87e1b3948953a9714f1cb573acd9447d6d6111ad8d188756407f7b2790c","Sense4FIT",false],["SUPER","53555045522d353037616136",18,"30450221008301715e7ddb7efb2cce9bf0ac48367a03b90bbd50e90453827033bb086feb680220716c749f95d178c12f30adc96bf395353a1dd37fbe57243abaa1c13a43d79e6a","SUPER",false],["USDC","555344432d633736663166",6,"3045022100f03b1ec0d83d8d75d436f688c2007ffb0c71b39a4f72aba036062ea8d7ade99402203449426b83203617279c04ccb00387a77deffc468c27015bfc1590f1950edde9","WrappedUSDC",false],["USDT","555344542d663863303863",6,"304502210094238f50b4f64f9b68ff8b38429ffc8bb0bd27ca21fdf758953843e4420cb273022010024029457a028b5a358c8f56b0dff27400321f7b999bd717844021d21fe77b","WrappedUSDT",false],["UTK","55544b2d326638306539",18,"304402204f2f6035dccb8c5e16698a1a8cfd0c0d4203e9a31103147e286fa1c1ea829c16022003284c1f3c8cb23b6587c5dee3187feb1e0663b15ffbffa89dbe07e7a766eff2","Utrust",false],["WAM","57414d2d353130653432",18,"3044022008f62bace38ce4cea98376818e519f6e03ba770e939c799117af8445de6125f7022020c1870a6c5685e8bf8ef0713c1b8dbe19faaf43bee8e66a63dd0fcfb0b0a6ca","WAM",false],["ZPAY","5a5041592d323437383735",18,"3045022100c550c564fa528e4087296b00cbf8d71262905e86fa61707d654013ac2b9f613f02202e28ea538cb0dcef9c65c2d9d22991604dd5b1f29a19365943c49f41a8c36eb0","ZoidPay",false]]
```

Comment:
Can you validate all tokens signatures and not only ride ? @lambertkevin spotted other failing tokens (example : AERO)